src/util/oilParser.mly

```diff
@@ -176,7 +176,3 @@ attribute_value:
   | OIL_STRING   {String $1}
   | AUTO         {Auto}
 ;
-
-%%
-
-let parse_error s = print_endline s
```

Comment:
Are you sure this is not a false positive? It seems to be the most recent definition of `parse_error`?

Reply:
It's not. I also wondered about this and indeed `parse_error` is a special thing you can define for ocamlyacc, but that has to be done in the header, not the trailer. I looked at the ocamlyacc output for this and it puts the default `parse_error` into its data structure before this definition, so it was completely useless.

I thought about moving this to the header, but since it never was there, nothing could possibly rely on it, so removing gives equivalent behavior. Not sure if actually overriding it wouldn't do something worse.